### ISSUE TITLE
FixedPagerAdapter gets broken by AndroidX jetification

### ISSUE THREAD

**Issue by lfielke** [NextFaze/power-adapters#59]
The `FixedPagerAdapter` class in module `power-adapters-support-v4` is put in the support library package `android.support.v4.view` in order to override `setViewPagerObserver`, which has default visibility. Unfortunately this does not survive the AndroidX jetifier, which rewrites the package to `androidx.core.view` because `FixedPagerAdapter` doesn't match any more specific rewrite rules. So when using AndroidX, it's no longer in the same package as `PagerAdapter`.

This means that `FixedPagerAdapter` either no longer overrides that method (on Android >= 5) or throws (on KitKat):
```IllegalAccessError: tried to access method androidx.viewpager.widget.PagerAdapter.setViewPagerObserver:(Landroid/database/DataSetObserver;)V from class androidx.core.view.FixedPagerAdapter```

Looking at the jetifier rules, there's one using a regex:
```
      "from": "android/support/v4/view/PagerAdapter(.*)",
      "to": "androidx/viewpager/widget/PagerAdapter{0}"
```

So a quick fix might be to rename `FixedPagerAdapter` to `PagerAdapterFixed` or similar so it matches that regex and gets jetified to the same package as the AndroidX `PagerAdapter` class.

A "proper fix" would be to make a new module for the AndroidX ViewPager.

**Comment by btilbrook-nextfaze** [comment]
> A "proper fix" would be to make a new module for the AndroidX ViewPager.

I wasn't planning on supporting the support library at all after migrating to AndroidX, so that seems about right.

**Comment by btilbrook-nextfaze** [comment]
Should be fixed in `0.21.0`

**Comment by lfielke** [comment]
Thanks, the fix in 0.21.0 works.
